src/components/FreeClass/ClassDetails/PageWrapper.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from "react"
+import { ScrollView, View, ViewStyle } from "react-native"
+import { NavBar, NavbarProps } from "components/NavBar"
+import { usePalette } from "utils/colors"
+
+/**
+ * Page Wrapper for Class Details page, maybe unify this with Settings Wrapper or any other wrapper
+ * somewhere in the app.
+ */
```

Comment:
sì direi che ha senso, non è che sia molto utile fare pagewrapper per ogni sezione ahah, andrebbero unificate, cercando di mantenere il resto del codice invariato e aggiungendo solo i mezzi per ottenere le differenze per ogni pagina, vedi tu se riesci a farlo in modo poco invasivo, se devi modificare troppa roba allora diventerà una issue futura

Reply:
Ok